src/ResourceManager/Profile/Commands.Profile/Environment/AddAzureRMEnvironment.cs

```diff
@@ -126,7 +126,9 @@ public override void ExecuteCmdlet()
                     newEnvironment.Endpoints[AzureEnvironment.Endpoint.ResourceManager] = ResourceManagerEndpoint;
                     newEnvironment.Endpoints[AzureEnvironment.Endpoint.ManagementPortalUrl] = ManagementPortalUrl;
                     newEnvironment.Endpoints[AzureEnvironment.Endpoint.StorageEndpointSuffix] = StorageEndpoint;
-                    newEnvironment.Endpoints[AzureEnvironment.Endpoint.ActiveDirectory] = ActiveDirectoryEndpoint;
+                    newEnvironment.Endpoints[AzureEnvironment.Endpoint.ActiveDirectory] =
```

Comment:
There is a test that is failing because of this in the check-in tests.  Also, you can use GeneralUtilities.EnsureTrailingSlash instead of custom code.